libs/mimir/src/adapters/secondary/elasticsearch/internal.rs

```diff
@@ -641,7 +642,10 @@ impl ElasticsearchStorage {
                 match result {
                     ElasticsearchBulkResult::Created => stats.created += 1,
                     ElasticsearchBulkResult::Updated => stats.updated += 1,
-                    _ => unreachable!("no port implements document deletion"),
+                    ElasticsearchBulkResult::NoOp => stats.skipped += 1,
+                    ElasticsearchBulkResult::Deleted => {
+                        unreachable!("no port implements document deletion")
```

Comment:
I think @woshilapin was expecting something like : 
```suggestion
                    ElasticsearchBulkResult::Deleted => {
                       stats.deleted +=1
```
Since this should not happen, I would also log a warning when it does

Reply:
Oh crap I was sure I did perform this increment, that's why I didn't understand the command :confused:

Reply:
That should be fixed [here](https://github.com/hove-io/mimirsbrunn/pull/760/commits/594176c04b60364cf351071a19ded98dbbefdd5d), it really didn't make any sense without it, sorry :sweat_smile: